pkg/deploy/uninstall.go

```diff
@@ -74,7 +74,6 @@ func (deploy *Deployer) uninstallMeteringCSV() error {
 	// and hope that the user is re-running the olm-uninstall command.
 	sub, err := deploy.olmV1Alpha1Client.Subscriptions(deploy.config.Namespace).Get(deploy.config.SubscriptionName, metav1.GetOptions{})
 	if apierrors.IsNotFound(err) {
-		deploy.logger.Warnf("The metering subscription does not exist")
```

Comment:
Why are we still bothering with the conditional?

Reply:
Not found is usually an acceptable condition during an uninstall since the intent is to remove the object.

Reply:
Aj, so the `IsNotFound` is trapping out the uninstall case. Should add a comment there to that effect then maybe.

Reply:
```go
// in the case where the subscription resource does not already exist, exit early
// and hope that the user is re-running the olm-uninstall command.
```

maybe not the clearest, but I think that's the intent 👍